rbac/common/base/base_state.py

```diff
@@ -247,46 +260,221 @@ def _find_in_state_container(self, container, address, object_id, target_id=None
         )
         return None
 
-    def get_from_state(self, state, object_id, target_id=None):
-        """Gets an address from the blockchain state
-        (state object is available when message is handled by transaction processor)"""
+    def deserialize(self, address, data):  # pylint: disable=unused-argument
+        """Deserialize the data of a blockchain address"""
+        # pylint: disable=not-callable
+        container = self._state_container()
+        container.ParseFromString(data)
+        return container
+
+    def get_address(self, context, address):
+        """Get the deserialized data of a blockchain address"""
+        data = state_client.get_address(context=context, address=address)
+        if data is None:
+            return None
+        return self.deserialize(address=address, data=data)
+
+    def get_addresses(self, context, addresses):
+        """Get the list of blockchain addresses"""
+        return state_client.get_addresses(context=context, addresses=addresses)
+
+    def get_from_state_context(self, context, object_id, target_id=None):
+        """Gets an address from the blockchain state"""
         address = self.address(object_id=object_id, target_id=target_id)
+        container = self.get_address(context=context, address=address)
+        if not container:
```

Comment:
Same logic here where 
if container:
return self_find_in_state_container(
            container=container,
            address=address,
            object_id=object_id,
            target_id=target_id,
        )

Reply:
R1710: Either all return statements in a function should return an expression, or none of them should. (inconsistent-return-statements)

Reply:
I did flip the condition around, but the statement must always explicitly return.